### ISSUE TITLE
Documentation for iptree?

### ISSUE THREAD

**Issue by pavel-odintsov** [zmap/iptree#5]
Hello!

I'm so excited by your project! Do you have any docs about it?

**Comment by dadrian** [comment]
No, but the only 4 functions are declared at https://github.com/zmap/iptree/blob/master/iptree.h on line 42.
